pygmt/helpers/testing.py

```diff
@@ -174,6 +174,7 @@ def download_test_data():
         "@hotspots.txt",
         "@ridge.txt",
         "@mars370d.txt",
+        "@MaunaLoa_CO2.txt",
```

Comment:
A few things:

- Move this up to L172 to be after `@EGM96_to_36.txt` so that we're sorting by ASCIIbetical order (capital letters before small letters). Maybe move up `@Table_5_11.txt` while you're at it.
- Add a comma to L170 to fix the problem mentioned at https://github.com/GenericMappingTools/pygmt/issues/1602#issuecomment-962676454
- Trigger the cache download!